.github/workflows/release.yml

```
@@ -0,0 +1,41 @@
+name: Release
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - '**'  # allow every tag including hierarchical tags like v1.0/beta
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to TestPyPI
+    runs-on: ubuntu-latest
```

Can we allow this job on failure?
```suggestion
    runs-on: ubuntu-latest
    continue-on-error: true
```

---

`continue-on-error` is step-specific. At job level, [there's still an issue open in GitHub](https://github.com/actions/toolkit/issues/399).
For the time being, you may not set this as required as a status check, maybe?

---

I understood from the github opened issue that `continue-on-error` is used for allowing the job to fail. However it's necessary to allow failure for TestRelease job since the release will definitely fail on the second try when using the same release version and to avoid changing the version back and forth while preparing for the main release.

---

i'm just quoting the reply of one of the collaborators of the `actions/toolkit` repository on the opened issue: https://github.com/actions/toolkit/issues/399#issuecomment-607450398(removed the mention to the original poster here)
> Hey, Thanks for the feature request!

>We do support marking a step to allow failure via continue-on-error.
We also support marking specific checks as required
That will allow you to merge pr's without a particular job succeeding.

> I think the latter should solve your issue at the job level, is there a reason that doesn't well for you?

So, basically, from what I understand here we can't mark the `continue-on-error` check at job-level, for that we should not make the status for this check as required. This will allow failures from this job to be allowed.

---

So let's make as not required.